### DISCUSSION TITLE
Farming taking 3x as long out of no where

### DISCUSSION THREAD

**Discussion by johnnyde94** [Chia-Network/chia-blockchain#5411] [Support]
Hello, I am not sure if this is a bug or an issues on my end but I seem to be having issues out of now where that is increasing my plot times. I have not changed any setting or hardware so I have not a clue what happened. Let me know what logs and info is needed to help solve this issue.

**Comment by lvcivs** [comment]
I assume you mean plotting, not farming (as in your title)?

Plotting time depends on a lot of factors, so it's hard to say anything without having more details about your setup, e.g. what OS are you using, did you install from git, are you plotting from GUI or CLI, what type of hardware etc.

**Comment by johnnyde94** [comment]
Yes my bad plotting, there were no hardware or software changes. It want from taking ~5 hrs to over 12 hours with the same exact settings and same number of parallel plots on the same type of hdd. 

John DeLuca

**Reply by ivanpavlina** [reply]
Check your temp ssd, could be that its dying and its struggling to 'find' healthy cells. If you are over or near rated TBW for it thats probably the problem.

**Reply by room101-dev** [reply]
On my system, my 1TB has a warranty TBW of 600TBW, I'm seeing NVME drives after plotting DIE at 150 TBW or 25% of warranty life
That's two weeks at my old plotting rate, now I plot 12/day
Don't expect an NVME to do more than 100 plots, its retire time, you will know.

Try running 'SUDO fstrim -v /dev/nvmeNNNN' and trim the drive, you should do that for every Nvme every 2-3 plots, once that doesn't even solve the problem, assume that NVME is dead, you can still use it for -2 temp, as sequential is ok, but -t requires random

If you run 'sudo nvme smart-log /dev/nvmeNNN' you can dump the stats if your busy-controller is high, it means your controller is dead that's the first to go, monitor temp always, early warning signs is +90C on the controller, then its toast.

The warranty at 600TBW for a 1TB samsung is normal usage, PLOTTING is something the devil created, its 1.5Trillion random read/writes over 8 hours, you do that 100 times and your NVME is dead, DOA ; like I said you can use it as -2 until it dies, best to try warranty if you can, but samsung is going to start refusing to honor warranty on mined drives, its easy on the log for them to see what you have been doing

**Reply by room101-dev** [reply]
> Check your temp ssd, could be that its dying and its struggling to 'find' healthy cells. If you are over or near rated TBW for it thats probably the problem.

Always log your NVME's on a standalone monitor bash routine, early warning is always +90C on the controller, its not the ram that dies, its the controller, say your doing 4x plots in parallel, the controller goes to 98C and is fried, but this usually happens around the 25% of NVME TBW warrranty range, I personally have little fans on all my NVME's now and keep them below 50C, but that's another story; all my machines have 3 nvmes, that way I can rotate out a failure without shutting down the machine to pullout the defective NVME, also a failed to plot NVME can still be use for -2 temp storage, how long I have yet to determine.

There are 100's of parameters here, not an easy thing to say just do this or this, study all the logs on 'sudo Nvme' linux cmd-line tool learn the data, known when the parms are signalling early death, if your NVME nears the 25% quit using it to plot, and redeploy it to pasture like we would do for an old horse.

People here talk a lot of crap about industrial NVME's and other non-sense, I have lots all kinds of stuff, nothing is designed to take this abuse, its been already said "CHIA-POS" was coded by the devil himself to destroy solid-state drives. Physical drives cannot take this abuse, I tried on both ssd-sata's and hdd's and in all cases the plots failed and the drives had to be re-formatted.

**Comment by johnnyde94** [comment]
Just did, I’m at 300TBW out of 1200TBW on my 980 Pro and 200TBW of 1200TBW on my 970 pro. I am also seeing an issue where Chia is not staying in sync despite having a solid internet connection. I checked for disconnects on my router and I found none.

John DeLuca

**Reply by ivanpavlina** [reply]
Then i have no idea, sorry.
About sync problems, yeah, there are alot of issues reported for that problem so i think we can only hope devs start to take that problem seriously

**Reply by MicroNateId** [reply]
> Just did, I’m at 300TBW out of 1200TBW on my 980 Pro and 200TBW of 1200TBW on my 970 pro. I am also seeing an issue where Chia is not staying in sync despite having a solid internet connection. I checked for disconnects on my router and I found none. John DeLuca
> […](#)
> On May 18, 2021, at 8:46 AM, Ivan Pavlina ***@***.***> wrote: ﻿ Check your temp ssd, could be that its dying and its struggling to 'find' healthy cells. If you are over or near rated TBW for it thats probably the problem. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

Check if your system Clock/time is synced with the UTC.  Having invalid clock time on your pc/server/etc can mess up sync.

**Reply by room101-dev** [reply]
25% is my rule of thumb, have u tried trim? If that doesn't work, and your stat's are high on the nvme smart-log, then your NVME is no longer able to plot, get over it, its how this scam works, its like the devil himself at samsung funded chia

**Comment by johnnyde94** [comment]
I ended up adding the chia inducer 
Eastern North America introducer-va.chia.net:8444
To fix hopefully my syncing issue. I think the slowness issue was due to an ssd of mine being on usb vers sata because I ran out of sata space. When I don’t use that usb ssd the plotting doesn’t slow down. It’s just odd that that caused all plots to slow vers just those plots in the ssd to slow.
John DeLuca

**Reply by rtsoliday** [reply]
I had a similar problem on Linux but it was because I wasn't mounting my SSDs with the discard option. After fixing that, it is running fast again.

**Reply by room101-dev** [reply]
> I had a similar problem on Linux but it was because I wasn't mounting my SSDs with the discard option. After fixing that, it is running fast again.

sure that will help the first time, but later you will need to run fstrim more&more frequently to keep up the speed, eventually you hit the 25% TBW warranty area, and even trimming  will not fix the problem, to know the problem always monitor "SUDO IOSTAT", it should always show 500MB for R/W, if it goes to zero, then that's a tell that the drive is NO longer suitable for plotting.

FYI, everybody should be formatting their drives with ZFS, and 'discard', but you still need to frequently trim after the first 10-20 plots,  discard is like telling the OS to trim once a day and/or at mount, but if your plotting 4 every 8 hours, then you need to trim every 8 hours.

The NVME drives boast of 5,000MB/s, but look carefully and its says 500mb/s random, and that is what CHIA-PLOTS are doing random, and not sequential, this is why its the controller that fails and not the ram, there are two thermometers on the NVME one for ram, and one for controller,you must monitor controller temp and be ready to suspend that plot if the temp exceeds threshold, now CHIA-POS should have put this shit in their code, but this is another story, every miner on earth has a max-temp switch to shutdown a gpu card when it goes over nnC

**Comment by BrandtH22** [comment]
@johnnyde94 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
